test/test-execute/exec-bind-paths.service

```diff
@@ -0,0 +1,14 @@
+[Service]
+Type=oneshot
+# Create a file in /tmp/test-exec_bind_paths
+ExecStart=/bin/touch /tmp/test-exec_bind_paths/thisisasimpletest
+# Then, the file can be access through /tmp
+ExecStart=/bin/sh -c 'test -f /tmp/thisisasimpletest'
+# Also, through /tmp/test-exec_bind_readonly_paths
+ExecStart=/bin/sh - c 'test -f /tmp/test-exec_bind_readonly_paths/thisisasimpletest'
```

Comment:
The space between `-` and `c` seems to make the command fail with `sh: 0: Can't open c`.

Reply:
Oh. Thanks.